docs/profiles.md

```diff
@@ -119,13 +121,17 @@ and to target all of these examples in a single `inspec.yml` file:
 ```YAML
 name: ssh
 supports:
-  - os-name: debian
-  - os-name: ubuntu
+  - platform-name: debian
+  - platform-name: ubuntu
     release: 14.04
-  - os-family: redhat
+  - platform-family: redhat
   - platform: aws
 ```
 
+With InSpec 2.0, we introduced the `platform` attribute, which allows to restrict the execution to `os`, `aws`, `azure` or `gcp`
```

Comment:
Do we actually have an attribute literally named `platform`?  This might be confusing

Reply:
Yes we do have `platform` which under the hood is if platform-name OR platform-family are true. I'll re-write this to be a bit more clear.